modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/inputs/DateTimeInput.es.js

```diff
@@ -12,29 +12,33 @@
  * details.
  */
 
+import ClayDatePicker from '@clayui/date-picker';
 import dateFns from 'date-fns';
 import propTypes from 'prop-types';
 import React from 'react';
 
-import {PROPERTY_TYPES} from '../../utils/constants.es';
-
-const INPUT_DATE_FORMAT = 'YYYY-MM-DD';
+const INPUT_DATE_FORMAT = 'YYYY/MM/DD';
```

Comment:
Why do you use this input date format for visualizing the dates? In Figma file I see the format YYYY-MM-DD and looks like the default format for the ClayDatePicker too.

Reply:
Ohh last time I check it was with '/'.. I'll change it